MovieChecker/MovieChecker/MovieDetailViewController.m

```diff
@@ -0,0 +1,34 @@
+//
+//  MovieDetailViewController.m
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "MovieDetailViewController.h"
+
+@interface MovieDetailViewController ()
+
+@property (nonatomic, weak) IBOutlet UILabel *movieTitle;
+@property (nonatomic, weak) IBOutlet UILabel *rating;
+@property (nonatomic, weak) IBOutlet UILabel *synopsis;
+@property (nonatomic, weak) IBOutlet UILabel *year;
```

Comment:
I would name these `...Label` so that the code reads more clearly—otherwise, they read like values rather than UI elements.